code_samples/data_migration/examples/create_attribute_group.yaml

```diff
@@ -21,12 +21,13 @@
         eng-GB: Color
     options:
         choices:
-            -    value: red
-	         label: 
-	             "eng-GB": "Red"
-            -    value: white
-	         label: 
-	             "eng-GB": "White"
-            -    value: black
-	         label: 
-	             "eng-GB": "Black"
+            -   value: red
+	            label: 
+	                "eng-GB": "Red"
+            -   value: white
+	            label: 
+	                "eng-GB": "White"
+            -   value: black
+	            label: 
+	                "eng-GB": "Black"
+
```

Comment:
These seem accidental and should be reverted back. Otherwise they are not valid YAML.

Reply:
Yes, this must be editor messing with indentation